velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
+* Update :doc:`Mathematical Functions <../functions/math>`.
+* Update :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide with new registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide with complex types.
+
+
+Core Library
+------------
+
+* Fix a bug in HashJoin filter to enable TPC-H query 19.
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add support for 'timestamp with timezone'.
+* Add support for masks to StreamingAggregation operator.
+* Add support for SSD-file data cache.
+* Other bug-fixes and code improvements.
+
+Presto Functions
+----------------
+
+* Add :func:`corr`, :func:`covar_samp`, :func:`covar_pop`, :func:`every` aggregate functions.
```

Comment:
I'm also seeing from_base and array_position being added in the following commits.

ac5f3df	Wei He	Mon Dec 20 11:36:53 2021 -0800	Add from_base Presto function (#776)
6378c84	Wei He	Wed Dec 15 17:15:28 2021 -0800	Add array_position Presto function (#671)

I use the following command to get commits for Dec:

```
git log --since='Dec 1 2021' --until='Dec 31 2021' --pretty=format:"%h%x09%an%x09%ad%x09%s" | grep -v "Updating submodules"
```

and this command to get a list of contributors for the Credits section:

```
git log --since='Dec 1 2021' --until='Dec 31 2021' --pretty=format:"%an" | grep -v svcscm | sort | uniq | tr '\n' ','
```

Reply:
`array_position` was added below in the `array functions` list. I will add `from_base`.